### ISSUE TITLE
Having issue with "avatars-avataaars-sprites" when using the base64 options giving characters outside of the Latin1 range error

### ISSUE THREAD

**Issue by gagan876574** [DiceBear/avatars#57]
InvalidCharacterError: Failed to execute 'btoa' on 'Window': The string to be encoded contains characters outside of the Latin1 range.
```
import Avatars from '@dicebear/avatars';
import sprites from '@dicebear/avatars-avataaars-sprites';
let options = {
    base64: true,    
  };
let avatars = new Avatars(sprites, options); 
let svg = avatars.create('beta');

```

**Comment by FlorianKoerner** [comment]
Should be fixed with version 4.0.5. Can you confirm this?

**Comment by gagan876574** [comment]
**It's fixed**. 
Working as **smooth** as **butter**.
